ext/opentelemetry-exporter-cloud-trace/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Add support for resources and resource detector
   ([#853](https://github.com/open-telemetry/opentelemetry-python/pull/853))
+- Improve resource and resource detector tests
```

Comment:
this probably doesn't need to be in the changelog. The changelog is intended to educate customers on behavioral changes.